projects/plugins/jetpack/_inc/client/my-plan/my-plan-header/index.js

```diff
@@ -343,6 +343,26 @@ class MyPlanHeader extends React.Component {
 					title: __( 'Jetpack VideoPress', 'jetpack' ),
 				};
 
+			case 'is-jetpack-social-basic-plan':
+				return {
+					...productProps,
+					details: [ activation, expiration ],
+					tagLine: __( 'Share your content to social media sites.', 'jetpack' ),
+					title: createInterpolateElement( __( 'Jetpack Social Basic', 'jetpack' ), {
+						em: <em />,
+					} ),
+				};
+
+			case 'is-jetpack-social-premium-plan':
+				return {
+					...productProps,
+					details: [ activation, expiration ],
+					tagLine: __( 'Share your content to social media sites.', 'jetpack' ),
```

Comment:
This partly why I'm not sure if we should include premium at this stage. I don't think we're in a position to write this tag line yet.